test/test_utils/test_sparql_queries.py

```diff
@@ -0,0 +1,110 @@
+from src.pandasaurus.utils.sparql_queries import (
+    get_contextual_enrichment_query,
+    get_curie_prefix_validation_query,
+    get_full_enrichment_query,
+    get_label_query,
+    get_minimal_enrichment_query,
+    get_obsolete_term_query,
+    get_replaced_by_query,
+    get_simple_enrichment_query,
+    get_slim_list_query,
+    get_slim_members_query,
+)
+
+
+def test_get_simple_enrichment_query():
+    s_iri_list = ["CL:0000084", "CL:0000787"]
+    o_iri_list = ["CL:0000788", "CL:0000798"]
+    property_list = ["rdfs:subClassOf"]
+
+    query = get_simple_enrichment_query(s_iri_list, o_iri_list, property_list)
+
+    expected_query = (
+        "SELECT ?s ?s_label ?p ?o ?o_label WHERE { GRAPH <http://reasoner.renci.org/redundant> {"
+        "VALUES ?s { CL:0000084 CL:0000787 } VALUES ?o { CL:0000788 CL:0000798 } VALUES ?p { rdfs:subClassOf }"
+        "?s ?p ?o. } ?s rdfs:label ?s_label. ?o rdfs:label ?o_label. FILTER(?s != ?o)}# LIMIT"
+    )
+
+    assert query == expected_query
+
+
+def test_get_minimal_enrichment_query():
+    pass
+
+
+def test_get_full_enrichment_query():
+    s_iri_list = ["CL:0000084", "CL:0000787"]
+    o_iri_list = ["CL:0000788", "CL:0000798"]
+
+    query = get_full_enrichment_query(s_iri_list, o_iri_list)
+
+    expected_query = (
+        "SELECT DISTINCT ?s ?s_label ?p ?o ?o_label WHERE { GRAPH <http://reasoner.renci.org/nonredundant> {VALUES ?s "
+        "{ CL:0000084 CL:0000787 } VALUES ?o { CL:0000788 CL:0000798 } ?s rdfs:subClassOf* ?o. FILTER(?s != ?o)} ?s "
+        "rdfs:label ?s_label. ?o rdfs:label ?o_label }# LIMIT"
+    )
+    assert query == expected_query
+
+
+def test_get_contextual_enrichment_query():
+    context_list = ["UBERON:0000362", "UBERON:0002113"]
+
+    query = get_contextual_enrichment_query(context_list)
+
+    expected_query = (
+        "SELECT * WHERE { VALUES ?context {UBERON:0000362 UBERON:0002113 } ?term BFO:0000050 ?context. ?term "
```

Comment:
should we handle property path `BFO:0000050*` query?

Reply:
Ubergraph is handling it. Everything is materialized in Ubergraph.

Reply:
The aim is to return all terms on a path from the query term up to the context term (in non-redundant graph).  For this I specified using the * operator on the NR graph.  Thinking about it, it may also be possible with redundant graph and direct queries, using conjunctive queries relating to query term and context term to set upper and lower bounds.  This might actually be safer as the non-redundant graph navigation my miss inferred relationships.